src/Datadog.Trace.ClrProfiler.Managed/Integrations/WcfIntegration.cs

```diff
@@ -145,11 +148,15 @@ private static Scope CreateScope(RequestContext requestContext)
                 scope = tracer.StartActive("wcf.request", propagatedContext);
                 var span = scope.Span;
 
+                object requestHeaders = requestMessage.GetProperty<object>("Headers").GetValueOrDefault();
```

Comment:
No need to change this, but for future reference if you only access properties from `requestHeaders` and don't need the value itself, you can chain `value.GetProperty(..).GetProperty(..).GetProperty(..)` without intermediate `GetValueOrDefault()`.

```suggestion
                object requestHeaders = requestMessage.GetProperty<object>("Headers");
```

Reply:
Gotcha. I wasn't sure what would be more performant: Calling `GetProperty` on the type `MemberResult<T>` instance (which is what the chain does) or calling `GetProperty` on the type `object` instance (which is what I ended up doing by calling `GetValueOrDefault` to return a result). Maybe we should benchmark the two of them.

Reply:
Good point.